bin/get-task-by-id

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+## This script getting SAPS task info by ID interacting with the Dispatcher backend.
+
+## checking args
+if [ $# -ne 4 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD TASK_ID DISPATCHER_ADDRESS"
+  exit 1
+fi
+
+# args
+readonly USER_EMAIL=$1
+readonly USER_PASSWORD=$2
+readonly TASK_ID=$3
+readonly DISPATCHER_ADDRESS=$4
+
+curl --silent --request GET "http://${DISPATCHER_ADDRESS}/tasks/${TASK_ID}" \
+  --header "Content-Type: text/plain"  \
+  --data-raw "userEmail=$USER_EMAIL&userPass=$USER_PASSWORD" | jq '.'
```

Comment:
why `jq '.'` ?

Reply:
I thought about leaving this for the user to have a better view of the script result, but I believe it can be removed

Reply:
I got your point now. For me it is ok :)